src/Configuration/test/CloudFoundry.Test/CloudFoundryServiceCollectionExtensionsTest.cs

```diff
@@ -49,4 +54,285 @@ public async Task ConfigureCloudFoundryOptions_ConfiguresCloudFoundryOptions()
         Assert.Equal(16384, appOptions.Value.Limits?.FileDescriptor);
         Assert.Equal("playground", appOptions.Value.SpaceName);
     }
+
```

Comment:
We should have a test using `TestWebApplicationBuilderFactory.CreateDefault()` to verify settings are bound from configuration (using both our own configurer, and the one from ASP.NET), and that the startup filter adds the middleware.